products/spanner/puppet.yaml

```diff
@@ -78,28 +78,9 @@ overrides: !ruby/object:Provider::ResourceOverrides
         )
     provider_helpers:
       - 'products/spanner/helpers/ruby/instance_helpers.rb.erb'
+  # Readonly resources.
   InstanceConfig: !ruby/object:Provider::Puppet::ResourceOverride
-    handlers: !ruby/object:Provider::Puppet::Handlers
```

Comment:
This still breaks existing configs, right?

Reply:
Oh, shoot. It does. One second.

Reply:
Fix made! No code changes should occur other than adding the deprecation message for those particular providers.